stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -253,4 +255,21 @@ public class ChannelListViewModel(
         activeChannelAction = null
         selectedChannel = null
     }
+
+    /**
+     * Uses [Channel.isMuted] property to store additional flag for each channel if the channel is muted
+     * for the current user.
+     *
+     * @param mutedChannelIds The list of channel IDs that represent channels muted for the current user.
+     * @return The list of augmented channels.
+     *
+     * @see [Channel.isMuted]
+     */
+    private fun List<Channel>.combine(mutedChannelIds: List<String>): List<Channel> {
```

Comment:
I would make this a regular function with two parameters and a more concrete name. 

This can be basically used with any `List<Channel>`and any `List<String>`. I know it's private, but still, the name doesn't say much.

I'd propose something like:

```
private fun enrichMutedChannels(channels: List<Channel>, mutedChannelIds: List<String>): List<Channel>{}
```

I know the signature above might be easier to use and all, but this would be much more clear and easier to understand if a person is coming from outside of this domain (e.g. new people)